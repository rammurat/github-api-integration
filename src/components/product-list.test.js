import React from 'react'
import renderer from 'react-test-renderer';
import { shallow, mount } from 'enzyme'
import { Provider } from 'react-redux'

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import Error from './error'

const store = {
    subscribe: () => { },
    dispatch: () => { },
    getState: () => ({})
}

const props = {
    labels: {
        a: 'b'
    }
}

it('renders correctly', () => {
    const tree = renderer
        .create(<Error />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});