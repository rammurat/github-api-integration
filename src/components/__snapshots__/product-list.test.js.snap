// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="product-list"
>
  <div
    className="row justify-content-end"
  >
    <div
      className="col-sm-12 col-md-6 col-lg-6"
    />
    <div
      className="col-sm-12 col-md-6 col-lg-6"
    >
      <nav
        aria-label="Page navigation example"
        className="product-pagination "
      >
        <ul
          className="pagination"
        >
          <li
            className="page-item"
          >
            <a
              className="page-link"
              href="#"
            >
              Previous
            </a>
          </li>
          <li
            className="page-item"
          >
            <a
              className="page-link"
              href="#"
            >
              1
            </a>
          </li>
          <li
            className="page-item"
          >
            <a
              className="page-link"
              href="#"
            >
              2
            </a>
          </li>
          <li
            className="page-item"
          >
            <a
              className="page-link"
              href="#"
            >
              3
            </a>
          </li>
          <li
            className="page-item"
          >
            <a
              className="page-link"
              href="#"
            >
              Next
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;
